### PR TITLE
Issue #1526 take care of pagination while fetching issue list

### PR DESCRIPTION
**Pull request by anjannath**
Fixes: #1526 

Added `per_page=100` in the api request, to return more data... should fix our issue

ref: https://developer.github.com/v3/#pagination

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@anjannath  I am running a different script to test it. Sorry about it. Can you fix the sprint-planning.sh in this PR?

**Comment by anjannath** [comment]
@LalatenduMohanty Sure.. let me have a look at it.

**Comment by LalatenduMohanty** [comment]
Looks like pagination works for sprint planning script too 

```diff
function github_milestone()
 {
-  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?milestone=$milestone\&state=all`"
+  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?per_page=100\&milestone=$milestone\&state=all`"

   csv_raw=`echo $milestone_data | jq --arg repo "$repository" '.[] | ";;;;;;;;;;;"
```

File: scripts/planning/sprint-planning.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ function usage()
 
 function github_milestone()
 {
-  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?milestone=$milestone\&state=all`"
+  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?per_page=100\&milestone=$milestone\&state=all`"
```
Comment (anjannath) [inline scripts/planning/sprint-planning.sh]: Decided not to use the token as there is more than one condition to handle, if the token given is wrong, if the token is empty.

**Pull request merged**